package.json

```diff
@@ -15,20 +15,14 @@
     "body-parser": "1.12.4",
     "connect-flash": "0.1.1",
     "connect-mongo": "0.8.1",
```

Comment:
MongoDB is good but I'm not too confident about using it with API integration >_<, not a blocker for web app completion. If we switch to mysql I will refactor it, so don't worry